ui/app/components/license-banners.js

```diff
@@ -31,8 +31,12 @@ export default class LicenseBanners extends Component {
 
   constructor() {
     super(...arguments);
-    // do not dismiss any banners if the user has updated their version
-    const dismissedBanner = localStorage.getItem(`dismiss-license-banner-${this.currentVersion}`); // returns either warning or expired
+    // do not dismiss any banners if the user has updated their license, which is indicated by a change in the this.args.expiry.
+    // do not dismiss any banners if the user has updated their version.
```

Comment:
I appreciate the extra context with these comments! The negative phrase `do not dismiss` was a little hard for me to follow at first - is this where we're 1) checking local storage and 2) comparing against the version + expiry date to determine whether or not to hide the banner? 

I may be misunderstanding, but if not then perhaps rewording to something like
```js
// reset and show a previously dismissed a license banner if:
// version or license (indicated by change in expiry date) are updated
```

Reply:
Got it. Updated, it's a little different than you're message. I'll change to
```
// reset and show a previously dismissed license banner if:
// the version has been updated or the license has been updated (indicated by a change in the expiry date).
```

Reply:
To me, I read your comment as the version is also indicated by a change from the expiry.

Reply:
sounds good!